packages/ethereum/hardhat.config.ts

```diff
@@ -71,22 +80,25 @@ function networkToHardhatNetwork(name: String, input: any): any {
   if (HOPR_HARDHAT_TAG) {
     cfg.tags = [HOPR_HARDHAT_TAG]
   }
-  if (cfg.tags.indexOf('development') >= 0) {
-    cfg.mining = {
+  if (cfg.tags && cfg.tags.indexOf('development') >= 0) {
+    ;(cfg as HardhatNetworkUserConfig).mining = {
       auto: true, // every transaction will trigger a new block (without this deployments fail)
       interval: [1000, 3000] // mine new block every 1 - 3s
     }
   }
   return cfg
 }
 
-const networks = {}
-
-const environment = PROTOCOL_CONFIG.environments[HOPR_ENVIRONMENT_ID] || {}
+const networks: NetworksUserConfig = {}
 
 for (const [networkId, network] of Object.entries(PROTOCOL_CONFIG.networks)) {
+  if (
```

Comment:
Previously behavior:
if one network had the `development` tag, then all `networks` get the `development` tag

Current behavior:
only set `developmennt` according to `protocol-config.json`